tests/DCPS/LivelinessTest/DataReaderListener.cpp

```diff
@@ -135,8 +133,8 @@ void DataReaderListenerImpl::on_liveliness_changed(
     last_status_ = status;
     ACE_DEBUG((LM_DEBUG,
       "(%P|%t) %T DataReaderListenerImpl::on_liveliness_changed %d\n"
-      "alive_count %d not_alive_count %d alive_count_change %d not_alive_count_change %d\n",
-      liveliness_changed_count_, status.alive_count, status.not_alive_count,
+      "%C alive_count %d not_alive_count %d alive_count_change %d not_alive_count_change %d\n",
+      liveliness_changed_count_, actor_.c_str(), status.alive_count, status.not_alive_count,
```

Comment:
If we're not incrementing `liveliness_changed_count_` anymore, we don't need to print the value here anymore.